### PR TITLE
fix: reduce size of the npm package

### PR DESCRIPTION
**Pull request by yhnavein**
Updated `.npmignore` file to exclude files that are not necessary in the bundle.

Package size difference:

**Before**:
```
npm notice package size:  3.8 MB
npm notice unpacked size: 16.1 MB
npm notice total files:   22
```

**After**:
```
npm notice package size:  246.8 kB
npm notice unpacked size: 1.0 MB
npm notice total files:   11
```

### CODE REVIEW

**Comment by yhnavein** [comment]
Bump

**Review by bcomnes — APPROVED** [review]
LGTM

**Comment by bcomnes** [comment]
I can't land until tests pass.  No idea why they are not running. Maybe try rebasing?

**Comment by bcomnes** [comment]
Damnit, I broke something on the test suite.  I will have to wait to get another approval done.

**Comment by bcomnes** [comment]
Ok, tests should work now.  Super sorry but can you rebase one more time?

**Comment by yhnavein** [comment]
Uhh

**Comment by bcomnes** [comment]
There we go! 

(tests require manual approval).

**Comment by bcomnes** [comment]
Going to try and get a release out this week.

**Comment by yhnavein** [comment]
Cheers and thanks! :)

**Pull request merged**
